README.md

```diff
@@ -174,4 +158,5 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 ## This toolbox is maintained by the
+
 [Microsoft Medical Image Analysis team](https://www.microsoft.com/en-us/research/project/medical-image-analysis/).
```

Comment:
Very weird way to do this. Maybe we can move that header into the text and use sth like "Maintenance" for the header.